docs/experimental-machine-learning/host-risk-score.md

```diff
@@ -7,33 +7,48 @@ To deploy this framework in your environment, follow the steps outlined below.
 
 #### 1. Unzip the release bundle
 
-#### 2. Navigate to the Dev Tools console in Kibana
+#### 2. Modify artifacts to reflect Kibana space
+For security reasons, we recommend that you restrict the Host Risk Score application to a specific Kibana space. In order to do so, run the `include_space.py` script with your Kibana space as the argument.
```

Comment:
`Recommend` is maybe too weak of a verb here; we _require_ the space to be accounted for in these scripts: 
```suggestion
For security reasons, we require that you restrict the Host Risk Score application to a specific Kibana space. In order to do so, run the `include_space.py` script with your Kibana space as the argument.
```